src/components/About/AboutDetails.module.css

```diff
@@ -0,0 +1,74 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+.container {
+  background: var(--midnight-green);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  max-width: 100%;
+  color: var(--primary-white);
+  padding-inline: 24px;
+  text-align: center;
+  gap: 16px;
+  position: relative;
+  padding-top: 80px;
+  padding-bottom: 108px;
+}
+
+.container::after {
+  content: "";
+  position: absolute;
+}
+
+.container::after {
+  width: 200px;
+  height: 100px;
+  bottom: 0px;
+  right: 0px;
+  transform: translate(50%, 0);
+  background-image: url("../../assets/bg-pattern-about-1-mobile-nav-1.svg");
```

Comment:
Background image should be on the container itself, not in a separate (pseudo-)element. This gets into z-index issues. Instead, use [background-position](https://developer.mozilla.org/en-US/docs/Web/CSS/background-position).